pkgs/by-name/ke/keepassxc-go/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "keepassxc-go";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "MarkusFreitag";
+    repo = pname;
```

Comment:
```suggestion
    repo = "keepassxc-go";
```

Reply:
Thank you, I missed that the variable was only used once.